json-smart/src/main/java/net/minidev/json/parser/JSONParserInputStream.java

```diff
@@ -54,13 +58,4 @@ public <T> T parse(InputStream in, JsonReaderI<T> mapper)
     //
     return super.parse(i2, mapper);
   }
-
-  //
```

Comment:
double check lines offset ans preserve some empty lines here probably should keep 5 out of 9